test/integration/confirmations/signatures/permit.test.tsx

```diff
@@ -0,0 +1,239 @@
+import { fireEvent, waitFor } from '@testing-library/react';
+import { ApprovalType } from '@metamask/controller-utils';
+import mockMetaMaskState from '../../data/integration-init-state.json';
+import { integrationTestRender } from '../../../lib/render-helpers';
+import { shortenAddress } from '../../../../ui/helpers/utils/util';
+import * as backgroundConnection from '../../../../ui/store/background-connection';
+import {
+  MetaMetricsEventCategory,
+  MetaMetricsEventName,
+  MetaMetricsEventLocation,
+} from '../../../../shared/constants/metametrics';
+import { MESSAGE_TYPE } from '../../../../shared/constants/app';
+import { createMockImplementation } from '../../helpers';
+
+jest.mock('../../../../ui/store/background-connection', () => ({
+  ...jest.requireActual('../../../../ui/store/background-connection'),
+  submitRequestToBackground: jest.fn(),
+}));
+
+const mockedBackgroundConnection = jest.mocked(backgroundConnection);
+const backgroundConnectionMocked = {
+  onNotification: jest.fn(),
+};
+
+const getMetaMaskStateWithUnapprovedPersonalSign = (accountAddress: string) => {
+  const pendingPermitId = 'eae47d40-42a3-11ef-9253-b105fa7dfc9c';
+  const pendingPermitTime = new Date().getTime();
+  return {
+    ...mockMetaMaskState,
+    preferences: {
+      ...mockMetaMaskState.preferences,
+      redesignedConfirmationsEnabled: true,
+    },
+    unapprovedTypedMessages: {
+      [pendingPermitId]: {
+        id: pendingPermitId,
+        status: 'unapproved',
+        time: pendingPermitTime,
+        type: MESSAGE_TYPE.ETH_SIGN_TYPED_DATA,
+        securityProviderResponse: null,
+        msgParams: {
+          from: accountAddress,
+          version: 'v4',
+          data: `{"types":{"EIP712Domain":[{"name":"name","type":"string"},{"name":"version","type":"string"},{"name":"chainId","type":"uint256"},{"name":"verifyingContract","type":"address"}],"Permit":[{"name":"owner","type":"address"},{"name":"spender","type":"address"},{"name":"value","type":"uint256"},{"name":"nonce","type":"uint256"},{"name":"deadline","type":"uint256"}]},"primaryType":"Permit","domain":{"name":"MyToken","version":"1","verifyingContract":"0xCcCCccccCCCCcCCCCCCcCcCccCcCCCcCcccccccC","chainId":1},"message":{"owner":"${accountAddress}","spender":"0x5B38Da6a701c568545dCfcB03FcB875f56beddC4","value":3000,"nonce":0,"deadline":50000000000}}`,
+          origin: 'https://metamask.github.io',
+          siwe: { isSIWEMessage: false, parsedMessage: null },
```

Comment:
`siwe` should not exist on permit messages.

Reply:
done, removed